cameronRomo/silver-nest
# Render residents components

### What’s this PR do?
- User is now able to view residents available for the program by semester in a scrollable grid of thumbnails
- User can click on a thumbnail and populate resident details
- Inline styles have been refactored to makeStyles object at start of component files
- Unused css/scss files have been removed from project
- Form and Thumbnails components are now named 'Form' and 'Thumbnails', respectively
- useState, useEffect hooks and variables were added to Thumbnails
### Where should the reviewer start?
- Profile.js, line 1
- FindARoommate, line 28, 29
### How should this be manually tested?
Open app in browser, click between tabs. Select semester in dropdown in FindARoommate tab. See resident thumbnails appear. Click on a resident to view more info in their Profile.
### Any background context you want to provide?
- Profile needs to be styled and potentially placed in a modal.
- Issues have been created for minor "Find A Roommate" button Route and Form-Select bugs 
### Screenshots (if appropriate):
![About Page](https://user-images.githubusercontent.com/68264128/104237707-ae490700-5415-11eb-9f7d-6eb79dae68cf.png)
![Find A Roommate + Selected Resident](https://user-images.githubusercontent.com/68264128/104237843-e6504a00-5415-11eb-926b-da518cff8fd0.png)
